CONTRIBUTING.md

```diff
@@ -78,41 +148,155 @@ Help us keep this project open and inclusive. Please read and follow our [Code o
 
 - Test your changes (create new tests as needed)
 
-- Submit a Pull Request
+- Once you’ve committed and pushed all of your changes to GitHub, go to the page for your fork on GitHub, select your development branch, and click the pull request button. Please ensure that you compare your feature branch to the desired branch of the repo you are supposed to make a PR to. If you need to make any adjustments to your pull request, just push the updates to GitHub. Your pull request will automatically track the changes in your development branch and update it. 🥳
+
+## (Option 2) Working on an issue - Make the changes via Locally
+
+Make sure you have the following prerequisites installed on your operating system before you start contributing:
+
+- [Nodejs and npm](https://nodejs.org/en/)
+
+  To verify run:
+
+  ```
+  node -v
+  ```
+
+  ```
+  npm -v
+  ```
```

Comment:
I don't know if these instructions are accurate -

Per the old CONTRIBUTING Doc

> Now that you have the code editor set up, you need to install the dependencies. To do this, you have to open the code editor's terminal and run the command bundle install.

Jekyll is built in Ruby so they would need to install ruby and then run `bundle install`

Reply:
The repository is ready for use with Codespaces or GitPod (Cloud Develpment). We use Devcontainer can read more in this post: [How Can I Contribute to the Black Python Developers Website using a development containers?](https://oleksis.github.io/jupyter/devcontainer/vscode/python/ruby/jekyll/2023/11/05/bpdevs-devcontainer-jekyll-python.html)

Reply:
Hi @kjaymiller, @oklesis, thanks for the feedback, will incorporate the changes accordingly.